src/hooks/useCheckbox.tsx

```diff
@@ -0,0 +1,10 @@
+import React, {useState} from "react";
+
+export const useCheckbox = (initialState: boolean):
+    [boolean, (event: React.ChangeEvent<HTMLInputElement>) => void] => {
+    const [isRemember, setIsRemember] = useState<boolean>(initialState)
+    const onChange = (_: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Переменную можно убрать
```suggestion
    const onChange = () => {
```

Reply:
Если я удалю эти переменные, то они не будут подходить под интерфейс IInput. Если я поменяю этот интерфейс, то если мне в будущем придётся использовать мой компонент input для чего-то, где мне нужны будут эти переменные, то мне придётся писать отдельный компонент input, который уже будет приниматься интерфейс, в котором эти переменные есть. Стоит ли?

Reply:
Сделай тогда параметр опциональным через `{ property?: type }`